test/evergreen.yml

```diff
@@ -262,14 +262,17 @@ functions:
           DEFINED_EVERGREEN_CONFIG_FLAGS=${DEFINED_EVERGREEN_CONFIG_FLAGS/\-DHAVE_BUILTIN_EXTENSION_ZSTD=1/}
         fi
 
-        if [[ "${build_variant|}" =~ "macos-13" ]]; then
+        if [[ "${build_variant|}" =~ "macos-14" ]]; then
           # For mac builds, we want explicitly tell cmake which python to use, as
           # well as the matching library directory and header files. The find_libpython
           # module gives us the library.
           SYSPY=${python_binary}
           $SYSPY -mvenv venv
           source venv/bin/activate
           pip3 install find_libpython==0.4.0
+          # FIXME-WT-13704: installing swig to accommodate for macos14 not having swig by default, will be removed once swig
```

Comment:
np: I would remove DEVPROD-12418 from the comment as it is internal to MongODB. The ticket WT-13704 should be enough and is public facing.

Reply:
sounds good, I've removed the devprod ticket number in: [6017ba5](https://github.com/wiredtiger/wiredtiger/pull/11174/commits/6017ba5be49b1189005fd3ed67153774adcb2932)